src/applications/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -14,83 +15,71 @@ import StepTwo from '../components/gpSteps/StepTwo';
 import StepThree from '../components/gpSteps/StepThree';
 import Warnings from '../components/gpMinorComponents/Warnings';
 
-export class GuidancePage extends Component {
-  constructor(props) {
-    super(props);
+export const GuidancePage = ({ formValues }) => {
+  const [accordionQuestionsState, setAccordionQuestionsState] = useState({
+    q1: false,
+    q2: false,
+  });
 
-    this.state = {};
-  }
+  useEffect(
+    () => {
+      // This effect hook only runs on mount OR if formValues dependency changes (Which is a new render/ or props to the component)
+      localStorage.setItem('dw-viewed-guidance', true);
+      localStorage.setItem('dw-formValues', JSON.stringify(formValues));
+      window.scrollTo(0, 0);
+    },
+    [formValues],
+  );
 
-  componentDidMount() {
-    localStorage.setItem('dw-viewed-guidance', true);
-    localStorage.setItem(
-      'dw-formValues',
-      JSON.stringify(this.props.formValues),
-    );
-
-    if (sessionStorage.getItem('dw-session-started')) {
```

Comment:
Same thing with these conditionals, not needed anymore?

Reply:
Yep, it wasn't doing another that the localstorage was already doing!